File: EjercicioPasanteHexacta/DTOS/PersonaDTO.cs
using EjercicioPasanteHexacta.Models;
using EjercicioPasanteHexacta.ViewModels;
using Microsoft.EntityFrameworkCore.Storage.ValueConversion.Internal;
using static System.Runtime.InteropServices.JavaScript.JSType;

namespace EjercicioPasanteHexacta.DTOS
{
    public class PersonaDTO
    {
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public int Edad { get; set; }
        public string EstadoCivil { get; set; }

        public PersonaDTO(string nombre, string apellido, int edad, string estadoCivil) {

            Nombre = nombre;
            Apellido = apellido;
            Edad = edad;
            EstadoCivil = estadoCivil;
        }

            
    }
}   

File: EjercicioPasanteHexacta/ClientApp/src/app/dtos/persona.dto.ts
export class PersonaDTO {
  public nombre: string;
  public apellido: string;
  public edad: number;
  public estadoCivil: string;

  constructor(nombre: string, apellido: string, edad: number, estadocivil: string) {
      this.nombre = nombre,
      this.apellido = apellido,
      this.edad = edad,
      this.estadoCivil = estadocivil
  }
}
